electrician/demo/sale_order_confirm.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+  <!--Update sale order stages-->
+  <function model="sale.order" name="action_confirm" eval="[[ref('sale_order_10'), ref('sale_order_7'), ref('sale_order_5'), ref('sale_order_6'), ref('sale_order_3'), ref('sale_order_1')]]"/>
```

Comment:
As we discussed earlier. Either you define the subsequent project/tasks within the files, either you confirm the SO  and those are automatically generated. Since you don't do much with the generated tasks, you can maybe use this and remove the project/task from xml files.
But the current state is not correct as it create it twice